eppo-sdk-test/BanditClientTest.cs

```diff
@@ -54,33 +61,33 @@ private EppoClient CreateClient(IAssignmentLogger? logger = null)
         }
         var config = new EppoClientConfig("mock-api-key", logger)
         {
-            BaseUrl = _mockServer?.Urls[0]!
+            BaseUrl = mockeServer?.Urls[0]!
```

Comment:
```suggestion
            BaseUrl = mockServer?.Urls[0]!
```
Needs changing everywhere